pubsub/synth.py

```diff
@@ -168,6 +168,20 @@ def _merge_dict(d1, d2):
 )
 
 # document FlowControl settings in Python 3.5+
+
+# ... first remove any existing code that injects the docs
+s.replace(
+    "google/cloud/pubsub_v1/types.py",
+    r"if sys\.version_info >= \(3, 5\):",
+    "",
+)
+s.replace(
+    "google/cloud/pubsub_v1/types.py",
+    r"    FlowControl\..*?__doc__ = \(.*?\)",
+    "",
+    flags=re.DOTALL,
+)
+
```

Comment:
I don't understand why we would be removing docstring changes for `FlowControl` while adding them for `BatchSettings`.

Reply:
We are not removing them, we are replacing them with ... the very same docstrings. Every synth run added a new block identical block for _FlowControl_, which is fixed here.

Spotted when reviewing #8444

Reply:
Background: I originally assumed that synth is ran only once after the files are generated, but this does not seem to be the case, and is expected? Or do we maybe have an issue with the workflow?

Reply:
@plamut Looking above, the file `"google/cloud/pubsub_v1/types.py"` is *excluded* from the `s.move` call, which means we shouldn't be fixing it in `synth.py` at all:  we should just be checking it in directly.

Reply:
Oh .. that's new info to me, thanks. Will make the change right away.